journal/modernized/04-chapter-3.adoc

```diff
@@ -516,7 +517,7 @@ and the men were taken and punished by the magistrates.
 After I was clear of my service in that nation, I returned to Dublin,
 took shipping for Whitehaven, landed safe, went home and found my family well.
 
-On the 17th of the ninth month, 1702.,
+On the 17th of the ninth month, 1702.
```

Comment:
I think probably this should be a comma, instead of a period, based on what follows.